### DISCUSSION TITLE
how can i cancel the sending request when using supabase js library?

### DISCUSSION THREAD

**Discussion by saman1324** [supabase/supabase#5992] [Questions] [answered]
hello. i have a problem. i want to cancel the sending request when using supabase js library. is it possible? if yes, how can i do that? thanks for your help.

**Comment by GaryAustin1** [comment, marked as answer]
Is this what you are looking for?
https://supabase.com/docs/reference/javascript/select#aborting-requests-in-flight

**Reply by saman1324** [reply]
yes thats it! thank you.

**Reply by html-3** [reply]
Is this solution still viable for v2? The links no longer redirects to the [correct page](https://supabase.com/docs/reference/javascript/v1/select#aborting-requests-in-flight).

**Reply by GaryAustin1** [reply]
v2 way. https://supabase.com/docs/reference/javascript/db-abortsignal

**Comment by dhurbhat** [comment]
Is there a similar method for canceling using Flutter SDK?
